pkgs/top-level/python-packages.nix

```diff
@@ -4665,6 +4665,13 @@ in {
 
   libpyvivotek = callPackage ../development/python-modules/libpyvivotek { };
 
+  libpwquality = pipe pkgs.libpwquality [
+    toPythonModule
+    (p: p.overrideAttrs (super: { meta = super.meta // { outputsToInstall = [ "py" ]; }; }))
+    (p: p.override { enablePython = true; inherit python; })
+    (p: p.py)
```

Comment:
Please add a comment what this is doing.

Reply:
It's doing what the others are doing and none of those have comments

Reply:
Hmmm... I think it would be easier to understand if we just passed in enablePython and then set meta inside the package with lib.optional.

Reply:
Personally I think this is fine.